extensions/sql-migration/src/api/utils.ts

```diff
@@ -865,3 +865,21 @@ export async function getBlobLastBackupFileNamesValues(lastFileNames: azureResou
 	}
 	return lastFileNamesValues;
 }
+
+export function getCorrectArmEndpointForAccount(account: AzureAccount): string {
```

Comment:
Can you please use the existing hosts info from here
[https://github.com/microsoft/azuredatastudio/blob/cb3ce6f1fd9d2b1185c3a483941522c336c42649/extensions/azurecore/src/account-provider/providerSettings.ts#L6](https://github.com/microsoft/azuredatastudio/blob/cb3ce6f1fd9d2b1185c3a483941522c336c42649/extensions/azurecore/src/account-provider/providerSettings.ts#L6)